params/cluster.go

```diff
@@ -14,62 +14,3 @@ type Cluster struct {
 	MailServers     []string `json:"mailservers"` // list of trusted mail servers
 	RendezvousNodes []string `json:"rendezvousnodes"`
 }
-
-// Consult this list with http://fleets.status.im/.
-var clusters = map[string]func() Cluster{
```

Comment:
This list will at least live in status-react through https://github.com/status-im/status-react/issues/5739, but I guess we could create one config file per cluster somewhere in the repo for ease-of-use. I had done this already but just for the `eth.test` cluster in https://github.com/status-im/status-go/blob/feature%2F1180-simple-config/.vscode/node-config.json.

Reply:
Another option could be for `statusd` to download the list from fleets.status.im. wdyt?

Reply:
> Another option could be for statusd to download the list from fleets.status.im. wdyt?

We can have a bot or script that would periodically check that and send a PR if the values have changed. We can't do it on the fly due to predictable builds effort. 

However, we might create an option `-fleet=https://some.address/fleet.json`but I don't see how it can be useful now.

Reply:
yeah, I was thinking it would be part of the `make setup` target to update files in a `.gitignored` folder.